src/ingameWindows/iwMapGenerator.cpp

```diff
@@ -34,14 +34,15 @@
 enum
 {
     CTRL_PLAYER_NUMBER = 2,
-    CTRL_MAP_STYLE     = 3,
-    CTRL_MAP_SIZE      = 4,
-    CTRL_PLAYER_RADIUS = 6,
-    CTRL_MAP_TYPE      = 8,
-    CTRL_RATIO_GOLD    = 10,
-    CTRL_RATIO_IRON    = 12,
-    CTRL_RATIO_COAL    = 14,
-    CTRL_RATIO_GRANITE = 16
+    CTRL_MAP_STYLE,
+    CTRL_MAP_SIZE,
+    CTRL_PLAYER_RADIUS,
+    CTRL_MAP_TYPE,
+    CTRL_RATIO_GOLD,
+    CTRL_RATIO_IRON,
+    CTRL_RATIO_COAL,
+    CTRL_RATIO_GRANITE,
+    CTRL_LAST_ID // last enum value
```

Comment:
Ok this does also work but I meant it the other way round. So just for documentation: Use `CTRL_LAST_ID = 10` as the **first** value in the enum and all the named entries come afterwards. This way you can use id `1` to `10` for all unnamed controls (labels etc.). It is better than this solution as it is shorter as instead of `CTRL_LAST_ID + x` you just use `x`